scripts/linters/pre_commit_linter.py

```diff
@@ -510,30 +530,35 @@ def main(args=None):
 
     # Concurrency limit: 25.
     concurrent_task_utils.execute_tasks(tasks_custom, custom_semaphore)
+
     # Concurrency limit: 2.
     concurrent_task_utils.execute_tasks(
         tasks_third_party, third_party_semaphore)
 
-    all_messages = []
+    lint_messages = []
 
     # Prepare semaphore for locking mechanism.
     semaphore = threading.Semaphore(1)
 
     for task in tasks_custom:
         semaphore.acquire()
-        _get_task_output(all_messages, task, semaphore)
+        _get_task_output(lint_messages, task, semaphore)
 
     for task in tasks_third_party:
         semaphore.acquire()
-        _get_task_output(all_messages, task, semaphore)
+        _get_task_output(lint_messages, task, semaphore)
 
-    all_messages += codeowner_linter.check_codeowner_file(
+    lint_messages += codeowner_linter.check_codeowner_file(
         verbose_mode_enabled)
 
-    _print_complete_summary_of_errors(all_messages)
+    _print_complete_summary_of_lint_messages(lint_messages)
+
+    errors_stacktrace = concurrent_task_utils.ALL_ERRORS
+    if len(errors_stacktrace) != 0:
```

Comment:
`if not errors_stacktrace` is more pythonic. (Ditto below)

Reply:
I think it is `if errors_stacktrace`